meerk40t/core/elements/offset_mk.py

```diff
@@ -368,10 +368,18 @@ def offset_path(self, path, offset_value=0):
         interpolation=500,
     )
     if p is None:
-        p = path
+        return path
+    g = Geomstr.svg(p)
+    # We are already at device resolution, so we need to reduce tolerance a lot
+    # Standard is 25 tats, so about 1/3 of a mil
```

Comment:
This differs between devices. Galvos can have a lot more precision depending on their lenses. Though this is just 0.1 native units or so, so it's fine.